tasks/main.yml

```diff
@@ -58,6 +58,16 @@
   notify:
     - restart keepalived
 
+- name: Configure /etc/sysconfig/keepalived parameters
+  template:
+    src: sysconfig_keepalived.j2
+    dest: "{{ keepalived_sysconfig_file_path }}"
```

Comment:
This is likely not enough:
1) The ubuntu init script won't load the generated file in defaults under ubuntu 14.04
2) The ubuntu init script crashes when giving this under ubuntu 16.04
3) The centos works.

Please adapt to restrict to the appropriate OS, or fix it under the other OSes, please